.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: rust
+cache: cargo
+
+os:
+  - linux
+  - osx
+
+rust:
+  - stable
+  - beta
+  - nightly
+
+before_script:
+  - rustup component add rustfmt
+
+script:
+  - cargo build --locked
+  - cargo test --locked
+  - cargo fmt --all -- --check
```

Comment:
I'm not planning to use rustfmt's formatting. It's a dumb tool that applied unconditionally sometimes makes formatting worse and less readable.

Reply:
hmm, did not know that before. 😄 

I will remove it.

Reply:
Just removed it.

Now it is just simple and easy build/test. :)